### DISCUSSION TITLE
Is there a way to allow all guests to submit change requests and comment?

### DISCUSSION THREAD

**Discussion by natanfudge** [GitbookIO/community#379] [0. General bugs and questions]
Hi, 
I'm evaluating Gitbook for my project.
One important feature I want to have is for arbitrary people reading the docs to be able to submit change requests. This would mean that if anything is outdated, unclear, or inaccurate, "the mass" could spot the issues and suggest fixes. 
This seems like a pretty basic feature, but I could not figure out how to do it. Is it possible?

**Comment by aikoii** [comment]
Hey @natanfudge, someone asked a question similar to yours and it was answered on this page 🙂 #267
